example/src/main/java/io/netty/example/http2/helloworld/server/Http2Server.java

```diff
@@ -69,12 +69,11 @@ public static void main(String[] args) throws Exception {
             sslCtx = null;
         }
         // Configure the server.
-        EventLoopGroup bossGroup = new NioEventLoopGroup(1);
-        EventLoopGroup workerGroup = new NioEventLoopGroup();
+        EventLoopGroup group = new NioEventLoopGroup();
         try {
             ServerBootstrap b = new ServerBootstrap();
             b.option(ChannelOption.SO_BACKLOG, 1024);
-            b.group(bossGroup, workerGroup)
+            b.group(group)
```

Comment:
it depends... many people just use the same EventLoopGroup. As it is often wasteful to have one thread that only accepts. That said it really depends on the pattern of the connections etc.